.vscode/settings.json

```diff
@@ -1,7 +1,3 @@
 {
-    "eslint.validate": [
-        "javascript",
-        "javascriptreact",
-        "typescript"
-    ]
+  "eslint.validate": ["javascript", "javascriptreact", "typescript"]
```

Comment:
why are there formatting differences here?

Reply:
unifying formatting across project (one of the tools is making this happen) so there is less differences between during merges